### DISCUSSION TITLE
Please help figure out this

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#17310] [General]
<img width="743" alt="image" src="https://github.com/YetiForceCompany/YetiForceCRM/assets/18006681/d2b0423c-e92f-4662-8dbe-9e7990e729e0">
look, when I choose the account, then I choose contact below account, the popup window shows the recordist by account, works fine!

but when I choose the partner then I want to choose contact below partner, the popup window also shows recordist by account...not by partner.  although I have wrote codes  to the app_data/moduleHierarchy.php
'Shipments' => ['contactno' => ['Contacts' =>['partnerid' =>['parent_id']]]],  
but nothing works.

please help

**Comment by deepti4666** [comment]
If you change the "contact" field to a one to many field, this might work.

**Reply by newminder** [reply]
yes, I did.  

the 1st contact can be limited according to Account, but the 2nd contact can not be limited by Partner(it is also limited by account) as I expected.

**Reply by deepti4666** [reply]
Hi, not sure how this works, I have tried a few things & nothing works.
